### PR TITLE
Use our standard placeholder variable names

### PR DESCRIPTION
**Pull request by lornajane**
I noticed that the `.env-example` used `API_KEY` rather than `NEXMO_API_KEY` which the code seemed to expect, so I changed it to match.

### CODE REVIEW

**Comment by lornajane** [comment]
Got carried away and added a repo as well.  Could you take another look please?

File: sms/send-with-slim/index.php [hunk 1]
```diff
@@ -10,8 +11,8 @@
         return $response->withStatus(400)->write("No message provided");
     }
 
-    $client = new Nexmo\Client(new Nexmo\Client\Credentials\Basic(API_KEY, API_SECRET));
-    $text = new \Nexmo\Message\Text($args['number'], NEXMO_FROM_NUMBER, $body['text']);
+    $client = new Nexmo\Client(new Nexmo\Client\Credentials\Basic(NEXMO_API_KEY, NEXMO_API_SECRET));
+    $text = new \Nexmo\Message\Text($args['number'], FROM, $body['text']);
```
Comment (mheap) [inline sms/send-with-slim/index.php]: When `config.php` is included it iterates over `$_ENV` and defines a constant for each key. This is how `NEXMO_API_KEY` and `NEXMO_API_SECRET` have values.

Following that, `FROM` should be `FROM_NUMBER` as that's the name in `.env`

Comment (lornajane) [reply sms/send-with-slim/index.php]: I think it's `.env` that wants updating then, `FROM` is our standard placeholder.

**Comment by lornajane** [comment]
This commit changes the `.env-example` to use `FROM`, as our building blocks do.  I couldn't find any other examples that used it, they mostly hardcode the from.  Hopefully good to go now.

**Pull request merged**